archiveServiceLambda/index.ts

```diff
@@ -0,0 +1,5 @@
+import { handleDdbToS3Event } from 'fhir-works-on-aws-persistence-ddb';
+
+exports.handler = async (event: any) => {
+    await handleDdbToS3Event(event);
```

Comment:
can we open a new ticket to pass through a config value as opposed to the persistence-ddb parsing the archiveConfig

Reply:
That makes sense

Reply:
How do you see the Lamdbda function accessing the configuration? Passing it into the event handler function? Calling a method on the interface component? I am in the process of creating the ticket.

Reply:
I think the Lambda function would access the configuration via environment variable, and pass the configuration to handleDdbToS3Event function. Like this
```
exports.handler = async (event: any) => {  
    // parse string format archive config into Map<string, number> format.
    const ttlsInSeconds = utility.parseArchiveConfig(process.env.ARCHIVE_CONFIG)
    await handleDdbToS3Event(event, ttlsInSeconds);
}
```

Reply:
We talked a little more about this while you were out. I have opened CHP-251. I think I captured what was discussed. Pleas take a look and followup in the Jira issue. Thanks.